source/extensions/clusters/eds/eds.h

```diff
@@ -88,7 +89,6 @@ class EdsClusterImpl
 
   Config::SubscriptionPtr subscription_;
   const LocalInfo::LocalInfo& local_info_;
-  const std::string cluster_name_;
```

Comment:
Can `cluster_name_` be kept as a reference to the `info_->name()` or `info_->edsServiceName()`?
(IIUC info is const and exists in the parent class so it should outlive the more specific class).

Reply:
It could keep a reference to the optional `edsServiceName` (and `info->name()` returns a `string` referene), but I don't think there is much benefit to this, since it's not on a hot path.